stl/inc/scoped_allocator

```diff
@@ -230,7 +230,16 @@ public:
 
     template <class _Ty, class... _Types>
     void construct(_Ty* _Ptr, _Types&&... _Args) { // construct with varying allocator styles
+#if _HAS_CXX20
+        _STD apply(
+            [_Ptr, this](auto&&... _New_args) {
+                _Scoped_outermost_traits<scoped_allocator_adaptor>::construct(
+                    _Scoped_outermost(*this), _Ptr, _STD forward<decltype(_New_args)>(_New_args)...);
+            },
+            uses_allocator_construction_args<_Ty>(inner_allocator(), _STD forward<_Types>(_Args)...));

```

Comment:
I believe that `uses_allocator_construction_args` should be `_STD` qualified. Also occurs in `<xmemory>`.

Reply:
I was confused about that, because all of the function calls except these were `_STD` qualified and I thought that it might be intentional to allow extending `uses_allocator_construction_args` for user defined types.

Reply:
I suspect it was an oversight in the original PR. WG21-N4878 16.4.2.2 [contents]/3 "Whenever a name `x` defined in the standard library is mentioned, the name `x` is assumed to be fully qualified as `::std::x`, unless explicitly described otherwise. For example, if the *Effects:* element for library function `F` is described as calling library function `G`, the function `::std::G` is meant." applies by default, so special wording is needed to tell implementers to use ADL. For example, [cmp.alg]/1.2 is the Standard's way of saying "use ADL here".

Unless I'm missing something, I see no such wording in [allocator.uses.construction] or elsewhere for `uses_allocator_construction_args`.